packages/kit/src/components/TxAction/TxActionTransfer.tsx

```
@@ -557,6 +557,42 @@ function TxActionTransferDetailView(props: ITxActionProps) {
 
       const transferExtraElements: React.ReactElement[] = [];
 
+      if (swapInfo && swapInfo.swapRequiredApproves) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)